lib/src/endpoints/categories.dart

```diff
@@ -1,11 +1,11 @@
-part of spotify;
+part of '../../spotify.dart';
 
 /// Endpoint of browsing the categories `v1/browse/categories`
 class Categories extends Browse {
   @override
   String get _path => '${super._path}/categories';
 
-  Categories(SpotifyApiBase api) : super(api);
+  Categories(super.api);
```

Comment:
Huh, interesting!

Reply:
I really like the `super.param` syntax. It reduces code repetition and maintains type integrity throughout the class hierarchy.

Reply:
It's a nice shorthand for sure!